getting-started/deployment_guide.md

```diff
@@ -92,8 +92,30 @@ karmor log
 5. [Amazon Elastic Kubernetes Service (EKS)](../deployments/EKS)
 
 ## Prerequisites
-1. [K3s](../deployments/k3s)
-2. [MicroK8s](../contribution/microk8s)
+Setting alternative environments for KubeArmor
+
+<details>
+  <summary>To set up KubeArmor using K3s cluster</summary>
```

Comment:
```suggestion
  <summary>K3s</summary>
```

Can we have bullet points here somehow?

Reply:
https://github.com/kubearmor/KubeArmor/pull/606#issuecomment-1033787759

You can refer this comment to see how we can achieve bullet points here,

Reply:
Resolving as this might be what we want I guess . Let me know if I need to make some improvements !